spec/controllers/api/v1/user_roles_controller_spec.rb

```diff
@@ -36,4 +44,72 @@
       expect(parsed_body['user_role']['user_id']).to eq(user_role.user_id)
     end
   end
-end
\ No newline at end of file
+
+  describe "POST user_roles" do
+    context "Reviewer" do
```

Comment:
A test for other than `Reviewer` role would also be good

Reply:
as well as testing as a public user. To ensure the abilities are correctly set

Reply:
Actually, rather than testing Roles, it's probably better to test permissions, and separately we can test that the correct roles have the right permissions.

Scenarios:
- Authenticated with `can_manage_user_roles`
- Authenticated without `can_manage_user_roles`
- Not authenticated

That's what drives the rules